dCommon/Diagnostics.cpp

```diff
@@ -115,6 +115,17 @@ void GenerateDump() {
 }
 
 void CatchUnhandled(int sig) {
+	std::exception_ptr eptr = std::current_exception();
+	try
+    {
+        if (eptr)
+            std::rethrow_exception(eptr);
+    }
+    catch(const std::exception& e)
+    {
+        LOG("Caught exception: '%s'", e.what());
+    }
```

Comment:
We currently use same line curly braces for try and catch like
```
try {

} catch (...) {

}
```
and for 1 line if statements it is preferred
` if (cond) logic;`

Reply:
Done. Sorry for all the CI triggers :D